tests/StubsTest.php

```diff
@@ -377,15 +379,50 @@ private static function getDuplicatedFunctions(array $filtered): array
         $duplicatedFunctions = array_filter($filtered, function (PHPFunction $value, int|string $key) {
             if (str_contains($key, 'duplicated')) {
                 $duplicatesOfFunction = self::getAllDuplicatesOfFunction($value->name);
-                $sinceVersionOfInitialFunction = Utils::getDeclaredSinceVersion(
+                $functionVersions[] = Utils::getAvailableInVersions(
                     PhpStormStubsSingleton::getPhpStormStubs()->getFunctions()[$value->name]);
-                $sinceVersionsOfDuplicatedFunctions = array_map(function (PHPFunction $function) {
-                    return Utils::getDeclaredSinceVersion($function);
+                $functionVersions[] = array_map(function (PHPFunction $function) {
+                    return Utils::getAvailableInVersions($function);
                 }, $duplicatesOfFunction);
-                return !empty(array_search($sinceVersionOfInitialFunction, $sinceVersionsOfDuplicatedFunctions));
+                $twoDimensionalArray = [];
```

Comment:
This part is really hard to follow. Do we have the following structure here: `["php_version" => ["function1", "function2"]`?

Reply:
Initially, we get an array with versions that can look like [[7.1,7.2],[[7.1]]] so we want to convert it to unified form for further processing and that's why `$twoDimensionalArray = [];` is initialized to later fill it with arrays of versions to get array like [[7.1,7.2],[7.1]]

Reply:
What is the place that outputs `[[7.1]]`? It looks like it shouldn't and this flattening is a workaround.

Reply:
Ok. So here is a simple example.
<img width="432" alt="Screenshot 2020-12-29 at 23 17 52" src="https://user-images.githubusercontent.com/37301285/103311719-25e96000-4a2c-11eb-8846-2f32fab19a34.png">
But it's possible that there will be several duplicates 
Plain array comes from `Utils::getAvailableInVersions(PhpStormStubsSingleton::getPhpStormStubs()->getFunctions()[$value->name]);` and other from duplicates and it's required to convert them

Reply:
Let's do the following instead than:
```
array_push($functionVersions, array_values(array_map(function (PHPFunction $function) {
                    return Utils::getAvailableInVersions($function);
}, $duplicatesOfFunction)));```